pkg/validation/utils_test.go

```diff
@@ -0,0 +1,71 @@
+package validation
+
+import (
+	"testing"
+)
+
+func TestIsInt(t *testing.T) {
+
+	tests := []struct {
+		name       string
+		arg        string
+		wantResult bool
+	}{
+		{
+			name:       "Case 1: numeric string",
+			arg:        "1234",
+			wantResult: true,
+		},
+		{
+			name:       "Case 2: alphanumeric string",
+			arg:        "1234abc",
+			wantResult: false,
+		},
+		{
+			name:       "Case 3: string with numbers and character",
+			arg:        "12_34",
+			wantResult: false,
+		},
```

Comment:
Should add a testcase for `arg: "0xff"` to catch in case someone tries to use `ParseInt` instead of `Atoi`. Note also that underscores in IDs are not allowed by regex.